src/main/scala/org/clulab/habitus/utils/TsvPrinter.scala

```diff
@@ -34,11 +34,12 @@ class TsvPrinter(outputFilename: String) extends Printer {
         // ``variable`` and ``value`` we access the two through ``arguments`` attribute of the Mention class.
         m =>
            {
-            val varText = m.arguments(printVars.mentionType).head.text
-            val value = m.arguments(printVars.mentionExtractor).head
-            val valText = value.text
+            val variable = m.arguments(printVars.mentionType).headOption
```

Comment:
Perhaps in a different iteration I would add an Opt or maybe to this like variableOpt or maybeVariable.  Not everyone agrees: https://stackoverflow.com/questions/11887181/scala-naming-convention-for-options.

Reply:
I'd go for Opt to match other CLU Lab code, though.  There isn't a standard Maybe class in Scala (yet).  Sometimes one goes along with true and false.